go/libraries/doltcore/schema/alterschema/modifycolumn_test.go

```diff
@@ -61,12 +61,12 @@ func TestModifyColumn(t *testing.T) {
 		},
 		{
 			name:           "remove null constraint",
-			existingColumn: schema.NewColumn("id", dtestutils.IdTag, types.UUIDKind, true, schema.NotNullConstraint{}),
-			newColumn:      schema.NewColumn("newId", dtestutils.IdTag, types.UUIDKind, true),
+			existingColumn: schema.NewColumn("age", dtestutils.AgeTag, types.UintKind, false, schema.NotNullConstraint{}),
```

Comment:
I'd expect to see some tests here of tag reuse. Are there some I'm not seeing?

Reply:
Testing of the tag reuse in general seems a little bit light. It was a bit confusing toggling between parts 1 and 2 of this change, so I missed this on the first pass. But while you do a good job of making sure the values are converted properly, I would like to see more tests that confirm that tag reuse works as expected.